internals/models.py

```diff
@@ -970,6 +970,8 @@ def __notify_feature_subscribers_of_changes(self, is_update):
       new_val = getattr(self, prop_name, None)
       old_val = getattr(self, '_old_' + prop_name, None)
       if new_val != old_val:
+        if new_val == '' and old_val is None:
+          continue
         new_val = convert_enum_int_to_string(prop_name, new_val)
```

Comment:
However, there are some fields in the diff with the following old and new values:

```
<li>
     <b>all_platforms:</b> <br/>
     <b>old:</b> None <br/>
     <b>new:</b> False<br/>
</li><br/>
<li>
     <b>wpt:</b> <br/>
     <b>old:</b> None <br/>
     <b>new:</b> False<br/>
</li><br/>
<li>
     <b>prefixed:</b> <br/>
     <b>old:</b> None <br/>
     <b>new:</b> False<br/>
</li>
```
Do we have to skip over None vs False case also?

Reply:
Yes I think that would be best.  I did not see that happening on our staging server, but I don't see any downside to handling that case because the diffs that you pasted are not helpful to users.

Reply:
Since this is the last detail, I'll go ahead and merge this PR and send you a new tiny PR to make this last change.  That should allow you to resolve conflicts and dig into the Cloud NDB upgrade.

Reply:
Done. See PR #1364